doc/api/path.md

```diff
@@ -330,6 +330,15 @@ For example on POSIX:
 path.normalize('/foo/bar//baz/asdf/quux/..');
 // Returns: '/foo/bar/baz/asdf'
 ```
+*Note*: The `path.posix.normalize()` method will not attempt to convert \ 
+(Windows) to / (POSIX), as \ is not recognized by POSIX as a valid directory 
+separator.  
+
+For example:
+```js
+path.posix.normalize("\\some\\thing\\like\\this")
```

Comment:
Suggestion: include `..` in the sample path to show that those segments are not removed from the normalized path.